service/history/historyEngine.go

```diff
@@ -825,10 +823,7 @@ func (e *historyEngineImpl) QueryWorkflow(
 
 	scope := e.metricsClient.Scope(metrics.HistoryQueryWorkflowScope)
 
-	consistentQueryEnabled := e.config.EnableConsistentQuery() && e.config.EnableConsistentQueryByNamespace(request.GetRequest().GetNamespace())
-	if request.GetRequest().GetQueryConsistencyLevel() == enumspb.QUERY_CONSISTENCY_LEVEL_STRONG && !consistentQueryEnabled {
-		return nil, ErrConsistentQueryNotEnabled
-	}
+	queryConsistencyLevel := enumspb.QUERY_CONSISTENCY_LEVEL_STRONG
```

Comment:
Left this as never change local var here in case if we decide to bring it back one day.